evmbin/src/display/std_json.rs

```diff
@@ -178,17 +240,23 @@ impl<Trace: Writer, Out: Writer> trace::VMTracer for Informant<Trace, Out> {
 		Self::with_informant_in_depth(self, subdepth, |informant: &mut Informant<Trace, Out>| {
 			let info = ::evm::Instruction::from_u8(instruction).map(|i| i.info());
 			informant.instruction = instruction;
-			let trace_data = json!({
-				"pc": pc,
-				"op": instruction,
-				"opName": info.map(|i| i.name).unwrap_or(""),
-				"gas": format!("{:#x}", current_gas),
-				"stack": informant.stack,
-				"storage": informant.storage,
-				"depth": informant.depth,
-			});
-
-			writeln!(&mut informant.trace_sink, "{}", trace_data).expect("The sink must be writeable.");
+
+			// Reference: https://serde.rs/attr-skip-serializing.html
```

Comment:
Unclear (to me) comment.

Reply:
It was just a link to a reference but it's not important so i've removed it in commit 8ab7a78189efb3c318827eb873bd5f4c8df67e51